vnext/local-cli/runWindows/utils/vstools.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+
+function hasBlock(lines, block) {
```

Comment:
name this more explicitly? what kind of block are we talking about?

Reply:
Fixed.